tests/sentry/workflow_engine/migrations/test_0060_rename_azure_devops_action_to_vsts.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from sentry.testutils.cases import TestMigrations
+
+
+@pytest.mark.skip("Timeout failures—skipping these tests, which pass, to unblock migration.")
```

Comment:
Have we followed up on this? It seems bad that we can't run migration tests.

Reply:
@asottile-sentry is working on squashing the migrations and setting up a process to make it easier in the future. This will hopefully resolve the issues we have here